lib/3scale/backend/stats/partition_eraser_job.rb

```diff
@@ -0,0 +1,64 @@
+module ThreeScale
+  module Backend
+    module Stats
+      # TODO: from configuration
+      DELETE_BATCH_SIZE = 50
+      # Job for deleting service stats
+      # Perform actual key deletion from a key partition definition
+      class PartitionEraserJob < BackgroundJob
+        # low priority queue
+        @queue = :main
```

Comment:
I think this should be in the `stats` queue or a new one that goes after all the existing ones.

`Main` has `NotifyJob`s that keep updated usage statistics on the master account. That should have precedence over these new jobs which are responsible only for cleanup tasks.

Reply:
:+1: